src/main/java/software/sigma/bu003/internship/vedmid_andrii/service/PartService.java

```diff
@@ -0,0 +1,75 @@
+package software.sigma.bu003.internship.vedmid_andrii.service;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+import software.sigma.bu003.internship.vedmid_andrii.entity.Part;
+import software.sigma.bu003.internship.vedmid_andrii.exception.PartNotFoundException;
+import software.sigma.bu003.internship.vedmid_andrii.repository.PartRepository;
+
+import java.util.List;
+import java.util.Objects;
+
+@Service
+@AllArgsConstructor
+public class PartService {
```

Comment:
there are not tests for service

Reply:
Why? They are in java/software/sigma/bu003/internship/vedmid_andrii/service/PartServiceTest.java.